releasenotes/1.9.3.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: releasenotes
+title: Release Notes for Version 1.9.3
+date: 2016-11-23 12:00:00
+release_notes: true
+permalink: /releasenotes/1.9.3/
+---
+
+1. [Introduction](#intro)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+You can read the official announcement at [Apache jclouds 1.9.3 released](http://markmail.org/message/lyozqjxkcxm47zh5). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+This is a bugfix release. The following list shows the fixed issues and improvements:
+
+#### Bug
```

Comment:
Most of these have not the corresponding JIRA issues. How about rewording this to:

*This is a bugfix release. The following list shows the fixed issues and patches that are included in the release:*
*#### Bugs and patches*